WalletWasabi.Fluent/WizardPage.cs

```diff
@@ -0,0 +1,13 @@
+using Avalonia.Markup.Xaml.Templates;
+using Avalonia.Metadata;
+
+namespace WalletWasabi.Fluent;
+
+public class WizardPage
```

Comment:
Can we just do custom TemplatedControl and not introduce another pattern here?

Reply:
Then you don't need special array list in resources.

Reply:
Just to custom templated control based e.g. on ContentControl and add properties as avalonia props.

Reply:
Or even better we can do it like send workflow with view models and use navigation stack. Would be even simpler then new wizard controls.

Reply:
I'd prefer not to use standard navigation for this scenario because it's a **wizard** in essence, therefore we should be able to display an indicator with the current progress.

Eg. the indicator here.
![image](https://github.com/zkSNACKs/WalletWasabi/assets/3109851/150e3386-d23a-4d9e-9de0-e43a183cdb98)

This is not possible with regular stack-based navigation unless you introduce extra complexity.